website/events/api/calendarjs/views.py

```diff
@@ -24,7 +26,22 @@ def get_serializer_context(self):
 
     def get_queryset(self):
         start, end = extract_date_range(self.request)
```

Comment:
Maybe it is possible to use query properties for this? Thats what job used for sales.

https://pypi.org/project/django-queryable-properties/

Reply:
I could try this for checking the number of registrations, but user_reg is based on services.is_user_registered, so I do not think I can turn that into a property.

Reply:
For number of registrations, this does not work since looking up the property still does a separate query to count the number of participants.